examples/java-matter-controller/java/src/com/matter/controller/Main.java

```diff
@@ -15,23 +15,85 @@
  *   limitations under the License.
  *
  */
+
 package com.matter.controller;
 
 import chip.devicecontroller.ChipDeviceController;
 import chip.devicecontroller.ControllerParams;
+import com.matter.controller.commands.common.Command;
+import com.matter.controller.commands.common.CredentialsIssuer;
+import java.util.Arrays;
 
 public class Main {
+  private static void ShowUsage(Command[] commands) {
+    StringBuffer arguments = new StringBuffer();
+    StringBuffer attributes = new StringBuffer();
+
+    for (Command command : commands) {
+      arguments.append("    ");
+      arguments.append(command.getName());
+
+      int argumentsCount = command.getArgumentsCount();
+      for (int j = 0; j < argumentsCount; j++) {
+        arguments.append(" ");
+        arguments.append(command.getArgumentName(j));
+      }
+
+      arguments.append("\n");
+
+      if ("read" == command.getName() && command.getAttribute().isPresent()) {
```

Comment:
`==` shouldn't be used to compare strings -- instead `.equals()` is necessary (the former compares memory references, which in the case of interned strings *may* work but isn't reliable). 

`if ("read".equals(command.getName())`